src/main/java/com/example/music/backend/playlist/domain/Playlist.java

```diff
@@ -0,0 +1,38 @@
+package com.example.music.backend.playlist.domain;
+
+import com.example.music.backend.user.domain.User;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@Table(name = "playlists")
+public class Playlist {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+    private Long countVideos;
+
+    @ManyToOne
+    private User owner;
+
+    @ElementCollection
+    @CollectionTable(name = "VIDEOS")
+    @Column(name = "YOUTUBE_ID")
+    @OrderColumn
```

Comment:
Do you really need all of them?

Reply:
removed @OrderColumn annotation, it is not needed